### ISSUE TITLE
gdb: doesn't support core dumps from `dumper`

### ISSUE THREAD

**Issue by jeremyd2019** [msys2/MSYS2-packages#2756]
Steps to reproduce:
```
$ sleep 500 &
[1] 6742

$ cat /proc/6742/winpid
7596

$ dumper core 7596
[1]+  Done                    sleep 500

$ ls -lrt
-rw-r--r-- 1 adminuser None   2308704 Dec 22 21:25 core.core

$ gdb --core core.core
GNU gdb (GDB) 10.2
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-pc-msys".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word".
Traceback (most recent call last):
  File "<string>", line 3, in <module>
ModuleNotFoundError: No module named 'libstdcxx.v6.printers'
/etc/gdbinit:6: Error in sourced command file:
Error while executing Python code.
"/home/adminuser/core.core": Core file format not supported
(gdb) q
```

**Comment by jeremyd2019** [comment]
Interestingly, on i686 msys, gdb --core loads the core,  but says that it can't read memory.  I have to use `gdb --core core.core /usr/bin/sleep`.  But it does actually work there...

**Comment by jeremyd2019** [comment]
Unfortunately `dumper` doesn't seem to work on ARM64 (against x86_64 MSYS processes).

**Comment by jeremyd2019** [comment]
https://cygwin.com/pipermail/cygwin-patches/2021q4/011640.html
> I think you need gdb 11 (for [this patch set](https://sourceware.org/pipermail/gdb-patches/2020-August/171232.html), which is also in  cygwin's  gdb 10 package) to read x86_64 cygwin core dumps.

**Comment by jeremyd2019** [comment]
Fixed by #2760
